README.md

```diff
@@ -2,24 +2,67 @@
+
+* [/face_recognition](https://github.com/webmachinelearning/webnn-samples/tree/master/face_recognition): This directory contains examples of SSD MobileNet V2 Face and Face Landmark (SimpleCNN) model implementation.
+* [/facial_landmark_detection](https://github.com/webmachinelearning/webnn-samples/tree/master/facial_landmark_detection): This directory contains examples of SSD MobileNet V2 Face and Face Landmark (SimpleCNN) model implementation.
+* [/image-classification](https://github.com/webmachinelearning/webnn-samples/tree/master/image_classification): This directory contains examples demonstrating image classification using pre-trained models with WebNN.
+* [/lenet](https://github.com/webmachinelearning/webnn-samples/tree/master/lenet): This example showcases the LeNet-based handwritten digits classification by WebNN API.
+* [/nsnet2](https://github.com/webmachinelearning/webnn-samples/tree/master/nsnet2): This example shows how to implement the NSNet2 baseline implementation of a deep learning-based noise suppression model.
+* [/object-detection](https://github.com/webmachinelearning/webnn-samples/tree/master/object_detection): Samples showcasing object detection tasks using WebNN with pre-trained models.
+* [/rnnoise](https://github.com/webmachinelearning/webnn-samples/tree/master/rnnoise): This example shows the RNNoise baseline implementation of a deep learning-based noise suppression model.
+* [/selfie_segmentation](https://github.com/webmachinelearning/webnn-samples/tree/master/selfie_segmentation): This example demonstrates the MediaPipe Selfie Segmentation using the TFLite Web XNNPACK delegate and WebNN delegate, built by tflite-support.
+* [/semantic_segmentation](https://github.com/webmachinelearning/webnn-samples/tree/master/semantic_segmentation): This directory contains examples of how to implement the DeepLab V3 MobileNet V2, from TFLite models.
+* [/style-transfer](https://github.com/webmachinelearning/webnn-samples/tree/master/style_transfer): Explore examples that highlight the artistic possibilities of WebNN by applying style-transfer techniques to images.
+
+## Getting Started
+To get started, follow these steps:    
+1. Clone the repository to your local machine and navigate to it:
+ ```bash
+> git clone --recurse-submodules https://github.com/webmachinelearning/webnn-samples
+> cd webnn-samples & npm install
+> npm start
+```
+2. Navigate to the desired sample directory that you want to explore.
+3. Read the accompanying README.md file for the sample to understand its purpose, requirements, and implementation details.
+4. Follow the instructions provided in the README to set up the necessary dependencies and run the sample.
+5. Experiment with the code and explore how WebNN can enhance machine learning tasks in the browser, navigating to http://localhost:8080.
```

Comment:
We may need to emphase currently WebNN implementation only supports CPU backend on Chrome or Edge behind a flag: [#enable-experimental-web-platform-features](chrome://flags/#enable-experimental-web-platform-features)

Reply:
@Honry, I think @inkade may have no idea on this, we can add it by ourselves.